src/common/reducers/entities/index.js

```diff
@@ -12,6 +12,11 @@ export function entities(state = { snaps: {}, repos: {} }, action) {
     return merge({}, state, action.payload.entities);
   }
 
+  // TODO merge master - duplication of above, do we need to support both formats?
+  if (action.payload && action.payload.response && action.payload.response.entities) {
```

Comment:
One of the issues after merge is that `entities` from API may be directly in action `payload` (if fetch was done directly) or in `response` prop (if it went through CALL_API middleware).